pkg/operator/controller/dns_status.go

```diff
@@ -24,6 +25,7 @@ func (r *reconciler) syncDNSStatus(dns *operatorv1.DNS, clusterIP, clusterDomain
 		if err := r.client.Status().Update(context.TODO(), updated); err != nil {
 			return fmt.Errorf("failed to update dns status: %v", err)
 		}
+		logrus.Infof("updated %s %s/%s: old: %#v, new: %#v", dns.TypeMeta.Kind, dns.ObjectMeta.Namespace, dns.ObjectMeta.Name, dns, updated)
```

Comment:
DNSes are cluster-scoped, so `dns.ObjectMeta.Namespace` and the corresponding expando are redundant.

Reply:
> DNSes are cluster-scoped...

Thanks.  Fixed in 523c09b5 -> 9cfdebe1